sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -237,6 +240,11 @@ trait WindowExecBase extends UnaryExecNode {
               new UnboundedWindowFunctionFrame(target, processor)
             }
 
+          case ("AGGREGATE_LAST", _, UnboundedPreceding, UnboundedFollowing, _) =>
```

Comment:
This means that we can't use one frame to evaluate `last_value` and other aggregate functions.

TBH I'm not sure if it's worth extending the framework for such a small optimization.

Reply:
You mean I understand. It is not worthwhile to optimize the `last_value` alone in the framework.
I also feel that way. If we can find other functions that are the same as `last_value` in the future, then we can do it.